stograde/student/markdownify/markdownify.py

```diff
@@ -2,6 +2,8 @@
 
 import os
 from collections import OrderedDict
+
+from .add_remove_supporting import import_supporting, remove_supporting
```

Comment:
What do you think about calling this file `supporting`, instead of `add_remove_supporting`?